console/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+---
+services:
+
+  caddy:
+    image: lucaslorentz/caddy-docker-proxy:ci-alpine
+    container_name: caddy
+    ports:
+      - 80:80
+      - 443:443
+    environment:
+      - CADDY_INGRESS_NETWORKS=caddy
+    networks:
+      - caddy
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - caddy_data:/data
+    restart: unless-stopped
+    depends_on:
+      - autobase-console-ui
+      - autobase-console-api
+    labels:
+      caddy.email: ${EMAIL}
+
+  autobase-console-api:
+    image: autobase/console_api:latest
+    container_name: autobase-console-api
+    restart: unless-stopped
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp/ansible:/tmp/ansible
+    depends_on:
+      - autobase-console-db
+    environment:
+      - PG_CONSOLE_AUTHORIZATION_TOKEN=${AUTH_TOKEN}
+      - PG_CONSOLE_DB_HOST=autobase-console-db
+      - PG_CONSOLE_LOGGER_LEVEL=${PG_CONSOLE_LOGGER_LEVEL:-INFO}
+      - PG_CONSOLE_DOCKER_IMAGE=autobase/automation:latest
+    networks:
+      - autobase
+      - caddy
+
+  autobase-console-ui:
+    image: autobase/console_ui:latest
+    container_name: autobase-console-ui
+    restart: unless-stopped
+    labels:
+      caddy: ${DOMAIN}
+      caddy.route_0: /api/v1/*
+      caddy.route_0.reverse_proxy: autobase-console-api:8080
+      caddy.route_1: /*
+      caddy.route_1.reverse_proxy: autobase-console-ui:80
+    depends_on:
+      - autobase-console-api
```

Comment:
Same comment here, those `depends on`  are unnecessarily slowing down the start of the stack.

Reply:
I think it doesn't hurt since one component depends on the other, for example, there is no point in the API if the database is not running, and in the UI if the API service is not running.

When I tested it, the startup was quite fast, so I don't think the startup time is a problem.

Reply:
ok, so as a side note for people who are looking at using it in production, this is not recommended practice.

Reply:
@sbusso Would you like to make a change according to the recommendation?

Reply:
I removed the "depends_on" parameter